### PR TITLE
feat: Support Google Blockly theme

### PR DESCRIPTION
**Pull request by kencwlau**
According to the update of Google Blockly in April 2019, themes provide 
a customized way to define look and feel for blocks and categories.
Thus, this commit add support to the block and category styles.

Reference: https://developers.google.com/blockly/guides/configure/web/themes

### CODE REVIEW

**Comment by roroettg** [comment]
Thank you for your PR. I will review it as soon as i can.

File: projects/ngx-blockly/src/lib/ngx-blockly/models/category.ts [hunk 1]
```diff
@@ -46,8 +54,23 @@ export class Category {
...
+    set style(value: string) {
+        this._style = value;
+    }
+
     public toXML(): string {
-        let xml = `<category name="${this.name}" colour="${this.colour}"`;
+        let xml = `<category name="${this.name}" `;
+
+        if (this.style === undefined) {
+            xml += `colour="${this.colour}" `
```
Comment (roroettg) [inline projects/ngx-blockly/src/lib/ngx-blockly/models/category.ts]: Missing semicolon

File: projects/ngx-blockly/src/lib/ngx-blockly/models/category.ts [hunk 2]
```diff
@@ -46,8 +54,23 @@ export class Category {
...
+        } else {
+            xml += `categorystyle="${this.style}" `
```
Comment (roroettg) [inline projects/ngx-blockly/src/lib/ngx-blockly/models/category.ts]: Missing semicolon

**Review by roroettg — CHANGES_REQUESTED** [review]
Would you mind updating the README.md and write a short example on how to use your code.

It would also be nice if you could adress the linter issues in the code.

**Comment by kencwlau** [comment]
> @roroettg  I will update the Readme.md and improve the coding quality as you suggested within these days. Thank you for your review!

**Comment by kencwlau** [comment]
@roroettg I have fixed the semicolon issue and add the Style part in the README.md, please review the changes, thanks!

**Comment by roroettg** [comment]
Thank your for your changes. I am currently on vacation and I will review your changes within the next week.

**Comment by roroettg** [comment]
Sorry, for the delay. I reviewed and tested your pull request and everything looks fine. Thanks again for your contribution.

**Pull request merged**